### PR TITLE
Added Extent Report Functionality

### PR DESCRIPTION
**Pull request by shivam198smart**
Added extent report.

### CODE REVIEW

File: src/main/java/UITestFramework/retryLogic/ExtentReportR.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import com.aventstack.extentreports.reporter.ExtentSparkReporter;
+
+ 
+
+public class ExtentReportR 
+{
+    static ExtentReports extent;
+    public static ExtentReports extentReportGenerator()
+    {
+    String path= System.getProperty("user.dir") + "\\report\\result.html";
```
[the remark this answers is not in the document]
Comment (shivam198smart) [reply src/main/java/UITestFramework/retryLogic/ExtentReportR.java]: Done.

File: src/main/java/UITestFramework/retryLogic/TestListener.java [hunk 2]
```diff
@@ -19,6 +19,8 @@
```
[the remark this answers is not in the document]
Comment (shivam198smart) [reply src/main/java/UITestFramework/retryLogic/TestListener.java:22]: added.

File: src/main/java/UITestFramework/retryLogic/ExtentReportR.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
+package UITestFramework.retryLogic;
+
+import com.aventstack.extentreports.ExtentReports;
+import com.aventstack.extentreports.reporter.ExtentSparkReporter;
+
+ 
+
+public class ExtentReportR
```
[the remark this answers is not in the document]
Comment (shivam198smart) [reply src/main/java/UITestFramework/retryLogic/ExtentReportR.java]: Changed it to ExtentReportSetup

**Comment by shivam198smart** [comment]
I have made the changes

**Comment by shankybnl** [comment]
Thank you @shivam198smart for your contribution :) Your changes are now merged!

**Pull request merged**